pkg/monitor/querier.go

```diff
@@ -111,11 +111,19 @@ func (monitor *agentMonitor) GetLocalPodNum() int32 {
 	return int32(monitor.interfaceStore.GetContainerInterfaceNum())
 }
 
-func (monitor *agentMonitor) GetAgentConditions() []v1beta1.AgentCondition {
+func (monitor *agentMonitor) GetAgentConditions(ovsVersion string) []v1beta1.AgentCondition {
```

Comment:
I think just pass a boolean like ovsdbConnected to this func, rather than ovsVersion.

Reply:
OK, got it.